index.html

```diff
@@ -3,17 +3,29 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>TV Show Project | My Name (My GitHub username)</title>
+    <title>
+      TV Show Project | Babak Jahangiri (https://github.com/babakjahangiri)
+    </title>
     <link href="style.css" rel="stylesheet" />
     <link rel="icon" href="/favicon.ico" type="image/x-icon" />
   </head>
 
   <body>
-    <div id="root">
-    </div>
+    <header>
+      <select id="drdShows" class="drdShows">
+        <option value="0"> -- ALL SHOWS -- </option>
+      </select>
+      <select id="drdEpisodes" class="drdEpisodes">
```

Comment:
Great Babak, it is better to use names which make sense, like here I don't understand drdshows what it means and it makes confusion for who reads your codes